subscriptions/sales_flow/create_a_quotation.rst

```diff
@@ -0,0 +1,78 @@
+==============================================
+Create a quotation using subscription products
+==============================================
+
+Selling a digital product or service can give you instant gratification. However, you then have
+to work hard for a new customer to make a purchase. It costs time and money. By convincing customers
+to sign up for a subscription, you will maximize your income and streamline your cash flow. You can
+sell any type of product or service through a subscription business model.
+
+.. raw:: html
+
+   <div align="center" style="color:#AD5E99; font-size: 2rem ;margin: 20px 0"> <b>The only limit is
+   your imagination.</b> </div>
+
+Here is a scenario using **Odoo Subscriptions** to create a quotation including subscription
+products.
+
+Create your first quotation
+===========================
+
+Before creating your first quotation, be sure to check out our documentation on how to create and
+manage your own *Subscription templates*:
+:doc:`../../subscriptions/configuration/subscription_templates`
+and on how to add *Subscription products* to your *Subscription templates*,
+:doc:`../../subscriptions/configuration/subscription_products`.
```

Comment:
Why don't you use :doc:`label <path_to_doc_page>`?

Reply:
Beuh... I find it easier to retrace the content path with .. /.. . Antoine also told me to do that.

Reply:
I don't suggest you to remove the ../.. but to integrate your label (Subscription template for example) in :doc: :smile:

Reply:
Great to learn something new :) The changes are made :D